src/eftest/runner.clj

```diff
@@ -84,10 +84,14 @@
   ([vars opts]
    (let [start-time (System/currentTimeMillis)]
      (if (empty? vars)
-       (println "No tests found.")
+       (do
+         (println "No tests found.")
+         test/*initial-report-counters*)
        (binding [report/*context* (atom {})
                  test/report      (:report opts progress/report)]
          (test/do-report {:type :begin-test-run, :count (count vars)})
          (let [counters (test-all vars opts)
-               duration (- (System/currentTimeMillis) start-time)]
-           (test/do-report (assoc counters :type :summary, :duration duration))))))))
+               duration (- (System/currentTimeMillis) start-time)
```

Comment:
This is entirely pedantic, but I wonder if use of `(System/nanoTime)` makes sense here… on the off chance that NTP kicks in and a clock skews. Not a change for this PR, but maybe something to change in #5?